### PR TITLE
[GH-#456] adjust multinode and clean helper in makefile

### LINKED ISSUE #456: adjust multinode and clean helper in makefile

### PR DESCRIPTION
**Pull request by thepiwo**
closes #456

### CODE REVIEW

**Comment by thepiwo** [comment]
@nedelcho-delchev-tues @meivantodorov can you review?

File: rel/config.exs [hunk 1]
```diff
@@ -3,36 +3,20 @@ Path.join(["rel", "plugins", "*.exs"])
...
-  set cookie: :"secret2"
-  set vm_args: "./rel/dev2/vm.args"
-end
-
-environment :dev3 do
-  set dev_mode: true
-  set include_erts: true
-  set cookie: :"secret3"
-  set vm_args: "./rel/dev3/vm.args"
+  set vm_args: "./rel/dev_build/vm.args"
```
Comment (nedelcho-delchev-tues) [inline rel/config.exs]: Here you set vm.args but the folder and file are missing.

File: rel/config.exs [hunk 2]
```diff
@@ -3,36 +3,20 @@ Path.join(["rel", "plugins", "*.exs"])
...
 
 use Mix.Releases.Config,
-
-default_release: :default,
-default_environment: Mix.env()
-
+    default_release: :default,
+    default_environment: Mix.env()
 
 environment :dev do
```
Comment (nedelcho-delchev-tues) [inline rel/config.exs]: If we are using dev_build I don't think we need the following lines

**Comment by nedelcho-delchev-tues** [comment]
When I build a new dev node in the _build directory I get `dev` instead of `dev_build` and the node is not working

**Comment by thepiwo** [comment]
@nedelcho-delchev-tues I dont understand.
```
make dev-build
make dev-start
make dev-attach
```

leads to a running and attached `dev_build` node for me, could you provide me with steps to reproduce your problem?

**Comment by nedelcho-delchev-tues** [comment]
Now its working

**Pull request merged**